src/Yoti.Auth/Conversion.cs

```diff
@@ -19,7 +19,7 @@ public static string BytesToBase64(byte[] bytes)
         /// UrlSafe Base64 uses '-' instead of '+', and '_' instead of '/' so it
         /// can be passed as a url parameter without extra encoding.
         /// </summary>
-        public static string BytesToUrlsafeBase64(byte[] bytes)
+        public static string BytesToUrlSafeBase64(byte[] bytes)
```

Comment:
There is another method in this class which is called `BytesToBase64`. `BytesToUrlSafeBase64` converts `+` to `-` and `/` to `_`, so that it can be used in a URL, although it's not being called from anywhere currently, but the reverse method `UrlSafeBase64ToBytes` is.